woocommerce/woocommerce-android
# Analytics Hub:  Improve date range selector UI

Changelog
==========
Reorganize the Analytics Hub selector UI to match the iOS version and rename the internal UI naming pattern for better identification with the range Selection class.

Screenshots
==========
| Before  | After |
| ------------- | ------------- |
| ![Screenshot_20230110_001942](https://user-images.githubusercontent.com/5920403/211454553-8ddd11d7-270b-4a6b-b800-250d7ba2e06c.png) | ![Screenshot_20230110_001750](https://user-images.githubusercontent.com/5920403/211454539-56172165-011d-4c63-87a8-6f6bd85e4a8e.png) |

Disclaimer
==========
Since it will take a bunch of changes to achieve the complete scope, this PR is targeted to a feature branch called `improve/analytics-hub` that will aggregate everything, this way we can review piece by piece while keeping the production version intact.

How to Test
==========
1. Build the app and open the Analytics Hub. If possible, keep a second device/emulator running the Hub from the `trunk` branch
2. Verify that Hub still opens following the selected tab from the My Store view. E. g. `This Year` tab should open the Hub selected in the `Year to Date` range.
3. Verify that all options for date range still exist and are the same from the original Hub

Update release notes:
